Swift/Tests/ReplicatorTest+Collection.swift

```diff
@@ -84,39 +84,38 @@ class ReplicatorTest_Collection: ReplicatorTest {
         
         var config = ReplicatorConfiguration(database: self.db, target: target)
         
-        let conflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
+        let conflictResolver: TestConflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
             return con.remoteDocument
         })
         config.conflictResolver = conflictResolver
-        XCTAssertNotNil(conflictResolver.uid)
-        
+        XCTAssertNotNil(conflictResolver)
         
         var colConfig = collectionConfig(config, col: try defaultCollection())
-        XCTAssertEqual(conflictResolver.uid, (config.conflictResolver as! TestConflictResolver).uid)
-        XCTAssertEqual(conflictResolver.uid, (colConfig.conflictResolver as! TestConflictResolver).uid)
+        XCTAssertEqual(conflictResolver, (config.conflictResolver as! TestConflictResolver))
+        XCTAssertEqual(conflictResolver, (colConfig.conflictResolver as! TestConflictResolver))
         
         // Update replicator.conflictResolver
-        let conflictResolver2 = TestConflictResolver({ (con: Conflict) -> Document? in
+        let conflictResolver2: TestConflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
             return con.localDocument
         })
-        XCTAssertNotEqual(conflictResolver.uid, conflictResolver2.uid)
+        XCTAssertNotEqual(conflictResolver, conflictResolver2)
         config.conflictResolver = conflictResolver2
         
-        XCTAssertEqual(conflictResolver2.uid, (config.conflictResolver as! TestConflictResolver).uid)
+        XCTAssertEqual(conflictResolver2, (config.conflictResolver as! TestConflictResolver))
         colConfig = collectionConfig(config, col: try defaultCollection())
-        XCTAssertEqual(conflictResolver2.uid, (colConfig.conflictResolver as! TestConflictResolver).uid)
+        XCTAssertEqual(conflictResolver2, (colConfig.conflictResolver as! TestConflictResolver))
         
         // Update collectionConfig.conflictResolver
-        let conflictResolver3 = TestConflictResolver({ (con: Conflict) -> Document? in
+        let conflictResolver3: TestConflictResolver = TestConflictResolver({ (con: Conflict) -> Document? in
             return nil
         })
-        XCTAssertNotEqual(conflictResolver2.uid, conflictResolver3.uid)
+        XCTAssertNotEqual(conflictResolver2, conflictResolver3)
         colConfig = collectionConfig(config, col: try defaultCollection())
         colConfig.conflictResolver = conflictResolver3
         config.addCollection(try defaultCollection(), config: colConfig)
         
-        XCTAssertEqual(conflictResolver3.uid, (config.conflictResolver as! TestConflictResolver).uid)
-        XCTAssertEqual(conflictResolver3.uid, (colConfig.conflictResolver as! TestConflictResolver).uid)
+        XCTAssertEqual(conflictResolver3, (config.conflictResolver as! TestConflictResolver))
```

Comment:
`XCTAssertTrue(conflictResolver3 === config.conflictResolver)` This will compare the reference when there is no need for uid.